src/modules/common/components/RemoteContent.js

```diff
@@ -8,15 +8,17 @@ type PropsType = {
   dangerouslySetInnerHTML: {
     __html: string
   },
-  hosts: Array<string>,
+  hijackRegExp: RegExp,
   onInternLinkClick: string => void,
   centered: boolean
 }
 
+const HIJACK = /https?:\/\/(cms\.integreat-app\.de|web\.integreat-app\.de|integreat\.app)(?!\/[^/]*\/(wp-content|wp-admin|wp-json)\/.*).*/
```

Comment:
I'm a bit against using RegExps if this gets more complicated. For now it looks fine though.

Reply:
Hmm... rather a function, right? I will leave it for now.